src/app/dashboard/components.tsx

```
@@ -27,9 +27,11 @@ import { getUser } from "@/lib/users/data";
 import {
   Building2,
   FileText,
+  Group,
```

ESLint

[error] 'Group' is defined but never used.

(@typescript-eslint/no-unused-vars)